PyInstaller/utils/hooks/hookutils.py

```diff
@@ -29,7 +29,7 @@
 
 # These extensions represent Python executables and should therefore be
 # ignored when collecting data files.
-PY_IGNORE_EXTENSIONS = set(['.py', '.pyc', '.pyd', '.pyo', '.so', 'dylib'])
+PY_IGNORE_EXTENSIONS = set(['.py', '.pyc', '.pyd', '.pyo', '.so', '.dylib'])
```

Comment:
Do you think this little change could break one unit tests in the continuous integration?